images/dvcr-artifact/cmd/dvcr-cleaner/cmd/ls.go

```diff
@@ -0,0 +1,166 @@
+/*
+Copyright 2024 Flant JSC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/spf13/cobra"
+)
+
+var (
+	NamespaceFlag     string
+	AllImagesFlag     bool
+	AllNamespacesFlag bool
+	ImageRepoDir      = "/var/lib/registry/docker/registry/v2/repositories"
+)
+
+var LsCmd = &cobra.Command{
+	Use:   "ls",
+	Short: "List of virtual images",
+	Args:  cobra.MatchAll(cobra.ExactArgs(1), cobra.OnlyValidArgs),
+}
+
+var lsViCmd = &cobra.Command{
```

Comment:
looks almost identical to lsCviCmd. You can put the logic into a separate function